integration/e2e/e2e_test.go

```diff
@@ -211,10 +212,61 @@ var _ = Describe("EndToEnd", func() {
 			nwo.CommitChaincode(network, "testchannel", orderer, chaincode, testPeers[0], testPeers...)
 			nwo.InitChaincode(network, "testchannel", orderer, chaincode, testPeers...)
 
+			By("listing the containers after committing the chaincode definition")
+			getPackageID := func(c nwo.Chaincode) string {
+				fileBytes, err := ioutil.ReadFile(c.PackageFile)
+				Expect(err).NotTo(HaveOccurred())
+				hashStr := fmt.Sprintf("%x", util.ComputeSHA256(fileBytes))
+				return hashStr
+			}
+
+			modulePkgID := getPackageID(chaincode)
+			gopathPkgID := getPackageID(gopathChaincode)
+			filters := map[string][]string{
+				"name": {modulePkgID, gopathPkgID},
+			}
```

Comment:
Can we name this something like `initialContainerFilter` and use `upgradedChaincodeFilter` for the second set?

If not, nbd...

Reply:
Done.